python/cuml/tests/test_kernel_ridge.py

```diff
@@ -178,6 +178,7 @@ def array_strategy(draw):
 
 @given(kernel_arg_strategy(), array_strategy())
 @settings(deadline=None)
+@pytest.xfail('https://github.com/rapidsai/cuml/issues/5177')
```

Comment:
```suggestion
@pytest.skip('https://github.com/rapidsai/cuml/issues/5177')
```

This needs to be skipped since many seeds will make it not fail, so we don't expect it to fail always